tools/pebble_ftdi_custom_pids.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
What should this copyright header contain for new files going forward?

Reply:
Should I add this in this PR? I see it's approved but I wasn't sure if I should add this above the Apache license first.

Reply:
It can be solved later as soon as license it's written somehow